sanic/app.py

```diff
@@ -194,6 +194,10 @@ def route(
             strict_slashes = self.strict_slashes
 
         def response(handler):
+            if isinstance(handler, (tuple, list)):
```

Comment:
Can we add some comments here for explaining?

Reply:
Added a comment explaining it